src/index.js

```diff
@@ -82,7 +85,8 @@ class PubSubRoom extends EventEmitter {
     const seqno = Buffer.from([0])
 
     const msg = {
-      from: this._ipfs._peerInfo.id._idB58String,
+      to: peer,
+      from: this._ipfs._peerInfo.id.toB58String(),
```

Comment:
This comment is not for this line, but for the `if (!conn)` block above:
We should add `this._connections[peer] = conn` at the end of the block so that we have the connections when calling `leave()` in order to close the connection.

Reply:
@haadcode good catch!
Should be fixed by cbe2c3ad532913bd131028a3ac948d6b7e890f24